3기/LeeGaYeon/week1/page1/style.css

```
@@ -0,0 +1,87 @@
+body {
+  /* font */
+  font-family: Spoqa Han Sans Neo;
```

```suggestion
  font-family: "Spoqa Han Sans Neo";
```

폰트페밀리에는 따옴표를 꼭 감싸주세요!
지금처럼 폰트명에 공백이 들어갔는데 따옴표로 감싸지 않았다면 폰트는 적용되고 있지 않고 CSS가 깨지고 있을거예요 😢